components/tidb_query/src/batch/runner.rs

```diff
@@ -46,6 +47,11 @@ pub struct BatchExecutorsRunner<SS> {
     collect_exec_summary: bool,
 
     exec_stats: ExecuteStats,
+
+    stream_batch_row_limit: usize,
```

Comment:
prefer to give comments to these two variables describing their roles, considering that these two variables' names are very similar.

Reply:
A rename for stream_batch_row_limit globally may also be considered. e.g. stream_min_rows_each_iter